src/me/mrCookieSlime/Slimefun/Setup/ResearchSetup.java

```diff
@@ -94,7 +94,7 @@ public static void setupResearches() {
 	    Slimefun.registerResearch(new Research(89, "Night Vision Goggles", 10), SlimefunItems.NIGHT_VISION_GOGGLES);
 	    Slimefun.registerResearch(new Research(90, "Pickaxe of Containment", 14), SlimefunItems.PICKAXE_OF_CONTAINMENT, SlimefunItems.BROKEN_SPAWNER);
 	    Slimefun.registerResearch(new Research(91, "Hercules Pickaxe", 28), SlimefunItems.HERCULES_PICKAXE);
-	    Slimefun.registerResearch(new Research(92, "Saw Mill", 2), SlimefunItems.SAW_MILL);
+	    Slimefun.registerResearch(new Research(92, "Table Saw", 2), SlimefunItems.TABLE_SAW);
```

Comment:
This research should still include the SAW_MILL in my opinion.

Reply:
Idea was to use the same research id to give existing players access to the new machine.   If the old one remains in the research won't they waste xp unlocking a machine that ONLY works with one wood type?

Reply:
You can add multiple Items to the same Research.